src/formatting/formattable.rs

```diff
@@ -54,37 +102,40 @@ mod sealed {
 impl<'a> sealed::Sealed for FormatItem<'a> {
     fn format_into(
         &self,
-        output: &mut impl io::Write,
+        output: &mut impl fmt::Write,
         date: Option<Date>,
         time: Option<Time>,
         offset: Option<UtcOffset>,
-    ) -> Result<usize, error::Format> {
-        Ok(match *self {
-            Self::Literal(literal) => write(output, literal)?,
+    ) -> Result<(), error::Format> {
+        match *self {
+            Self::Literal(literal) => {
+                output.write_str(String::from_utf8_lossy(literal).as_ref())?
```

Comment:
This `from_utf8_lossy` call is where my assumption about utf-8 falls through

Reply:
Fixed in https://github.com/time-rs/time/pull/407/commits/959bb7399d3befee840af6f0338a886e87615468